mzLib/FlashLFQ/SpectraFileInfo.cs

```diff
@@ -39,5 +39,9 @@ public override int GetHashCode()
         {
             return FullFilePathWithExtension.GetHashCode();
         }
+        public override string ToString()
+        {
+            return FilenameWithoutExtension;
```

Comment:
don't know the context here from looking at files changed but keeping the extension helps with .raw/.mzML confusion.